src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/ImporterConfiguration.kt

```diff
@@ -27,5 +27,6 @@ class ImporterConfiguration {
   fun timeSource() = TimeSource { LocalDateTime.now(Clock.systemDefaultZone()) }
 
   @Bean
-  fun supplierPrices() = SupplierPrices { supplier -> priceRepository.findBySupplier(supplier) }
+  fun supplierPrices() =
+    SupplierPrices { supplier, year -> priceRepository.findBySupplierAndEffectiveYear(supplier, year) }
```

Comment:
This is used by the spreadsheet generation.  With the introduction of next years prices we now have to consider which effective prices are included in the generated spreadsheet i.e. we only include the prices we are interested in.